backend/Tasque.Core.Common/Entities/Abstract/IBaseEntity.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Tasque.Core.Common.Entities.Abstract
+{
+    public interface IBaseEntity
```

Comment:
For what do we need empty interface? I'm just wondering

Reply:
Unfortunately, it was impossible to implement what I wanted using the basic class. This interface is used to label our entities. This is convenient when we want to write the expression "class, IBaseEntity"

Reply:
Good point. I thought about it and decided to just use a base class